sound/soc/sof/Kconfig

```diff
@@ -166,6 +166,15 @@ config SND_SOC_SOF_DEBUG_RETAIN_DSP_CONTEXT
 	  Say Y if you want to retain DSP context for FW exceptions.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_PROBES
+	bool "SOF enable probes"
+	select SND_SOC_COMPRESS
+	help
+	  This option enables the data probing feature that can be used to
+	  gather data directly from audio stream.
```

Comment:
from specific points of the audio pipelines/graph.

Reply:
Ack.